rust_src/lib.rs

```diff
@@ -0,0 +1,336 @@
+use bincode::{deserialize, serialize};
```

Comment:
Is it possible to move the `rust_src` directory somewhere in the `river` directory?